### PR TITLE
add reasonable width to grade format button

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** 
Grade format button in Gradebook has too-short width, causing the caret icon to overlay the text.
Added a slight right-padding to fix.

Before:
image
After:
image

JIRA: [AU-30](https://openedx.atlassian.net/browse/AU-30)

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**
- [ ] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by jansenk — APPROVED** [review]

**Review by mattcarter — APPROVED** [review]
Padding? Right.  Approved.
